dao/src/main/java/com/epam/esm/criteria/certificates/DescriptionPartsGiftCertificateCriteria.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.esm.criteria.certificates;
+
+import com.epam.esm.constants.ApplicationConstants;
+import com.epam.esm.criteria.AbstractCriteria;
+import com.epam.esm.criteria.Criteria;
+import com.epam.esm.entity.GiftCertificate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcCall;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
```

Comment:
Please, avoid imports with .*

Reply:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

Reply:
commit 53f030d